src/ServerSide/InternalTypes.elm

```diff
@@ -105,6 +108,15 @@ decodeTagger =
         ]
 
 
+decodeKeyedNode : Json.Decode.Decoder NodeRecord
+decodeKeyedNode =
+    Json.Decode.object4 NodeRecord
+        ("tag" := Json.Decode.string)
+        ("children" := Json.Decode.list (Json.Decode.at [ "_1" ] decodeNodeType))
```

Comment:
@eeue56  funny thing! can't do it. this is the recursion bug thing I was talking about: https://github.com/eeue56/elm-server-side-renderer/pull/5#issuecomment-265248450